### PR TITLE
docs: remove experimentalSessionAndOrigin

### PR DESCRIPTION
**Pull request by mschile**
Documentation changes for https://github.com/cypress-io/cypress/pull/24340

### CODE REVIEW

File: content/api/commands/origin.md [hunk 1]
```diff
@@ -383,81 +354,6 @@ and reuse it across tests.
 
 ## Notes
 
-### Migrating existing tests
```
Comment (ryanthemanuel) [inline content/api/commands/origin.md:386]: Do we want to keep around the migration documentation? That seems like it might be still useful

Comment (ryanthemanuel) [reply content/api/commands/origin.md:386]: Yeah I think that makes sense. I just wanted to make sure this doesn't get lost.

Comment (mschile) [reply content/api/commands/origin.md:386]: Moved the content to a migration guide. [dde69c1](https://github.com/cypress-io/cypress-documentation/pull/4807/commits/dde69c1ebf2d547bc1f334b38734966b63196da2)

**Review by emilyrohrbough — APPROVED** [review]

File: content/_changelogs/11.0.0.md [hunk 2]
```diff
@@ -4,6 +4,9 @@ _Released MM/DD/YYYY_
 
 **Breaking Changes:**
 
+- `experimentalSessionAndOrigin` flag has been removed and all functionality is
+  on by default now.
+- `testIsolation` defaults to `strict` now.
```
Comment (AtofStryker) [inline content/_changelogs/11.0.0.md:9]: @emilyrohrbough is this `strict` or `on`? I guess this changes pending https://github.com/cypress-io/cypress/pull/24316?
```suggestion
- `testIsolation` defaults to `on` now.
```

**Review by AtofStryker — COMMENTED** [review]
Proposed changes look good to me. Guessing we need to update pending https://github.com/cypress-io/cypress/pull/24316 to reflect `strict` test isolation?

File: content/_changelogs/11.0.0.md (hunk 2 above)
Comment (emilyrohrbough) [reply content/_changelogs/11.0.0.md:9]: @AtofStryker This will be `on` once my changes merge. I'll have to update once that PR is merged

**Review by ryanthemanuel — APPROVED** [review]

**Review by AtofStryker — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**